kinetic/distribution.yaml

```diff
@@ -998,6 +998,22 @@ repositories:
       url: https://github.com/asmodehn/certifi-rosrelease.git
       version: 2015.11.20-3
     status: maintained
+  cht10_node:
+    doc:
+      type: git
+      url: https://github.com/Playfish/cht10_node.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/Playfish/cht10_node_release.git
+      version: 0.0.1-1
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.